lib/matplotlib/axes/_axes.py

```diff
@@ -4440,6 +4515,8 @@ def get_interp_point(ind):
             polys.append(X)
 
         collection = mcoll.PolyCollection(polys, **kwargs)
+        color_cycle = self._get_lines.color_cycle
```

Comment:
:-1: for similar reasons to above.  This is a major API break and if we _do_ want to do this it needs to be done on `color_overhaul`.